packages/firebase_core/android/build.gradle

```diff
@@ -34,12 +34,29 @@ rootProject.allprojects {
 
 apply plugin: 'com.android.library'
 
+/**
+ * Get the current version of the plugin from the plugin's pubspec.yaml file.
+ *
+ * @return String representing version of the plugin.
+ */
+def flutterFireVersion() {
+    File pubspec = new File('../../pubspec.yaml')
```

Comment:
This may be cleaner:

```groovy
File pubspec = new File(project.projectDir.parentFile,  'pubspec.yaml')
```

project.projectDir = `<pluginRoot>/android`
parentFile = `<pluginRoot>`

----

Otherwise should probably use `File.separator` for Windows support instead of `/`